core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -385,7 +399,24 @@ public String toString() {
         .add("lower_bounds", lowerBounds)
         .add("upper_bounds", upperBounds)
         .add("key_metadata", keyMetadata == null ? "null" : "(redacted)")
-        .add("split_offsets", splitOffsets == null ? "null" : splitOffsets)
+        .add("split_offsets", splitOffsets == null ? "null" : splitOffsets())
+        .add("equality_ids", equalityIds == null ? "null" : equalityFieldIds())
         .toString();
   }
+
+  private static int[] toIntArray(List<Integer> ints) {
```

Comment:
If the data class pattern in Iceberg is:
- stores (primitive) arrays in class field.
- provides List method to return this field.

Can we add `ArrayUtil` class to provide `toPrimitive` and `toList` utils?

Reply:
Moved to `ArrayUtil`.